aries_cloudagent/config/argparse.py

```diff
@@ -780,8 +780,7 @@ def add_arguments(self, parser: ArgumentParser):
     def get_settings(self, args: Namespace):
         """Extract mediation settings."""
         settings = {}
-        if args.mediation:
-            settings["mediation.open"] = True
+        settings["mediation.open"] = args.open_mediation
```

Comment:
I assume this will default to false if the flag is absent?

Reply:
I believe so. If you do not provide a flag the args structure will never get the attribute `open_mediation`. I saw this used in another part of the arg parser and followed the same pattern for continuity.